modules/ROOT/pages/tuning-test-validations.adoc

```diff
@@ -0,0 +1,85 @@
+= Performance Test Validations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To start validating your application's performance, first understand the prerequisites to testing, follow the recommended best practices, and then execute performance testing.
+
+
+== Prerequisites to Perform Test Validations
+
+Before testing the performance, consider the following points:
+
+* Confirm that your Mule app and its functions work as expected because a wrong flow can give false-positive data.
+* Establish the performance criteria for the test cases to determine the goal and scope for tuning. The following questions can help you clarify the performance requirements:
+** What are the expected average and peak workloads?
+** Is the use case emphasis on throughput, concurrency, response time or size of messages? Some common cases are:
+*** Focus on throughput when handling a large volume of transactions is a high priority.
+*** Focus on response time or latency if spikes affect user experience and becomes frustrated.
+*** Focus on concurrency if it is necessary to support a high number of users connecting at the same time.
+*** Focus on managing large messages when the application is transferring, caching, storing, and or processing a payload bigger than 1 MB.
+** What is the minimum acceptable throughput?
+** What is the maximum acceptable response time?
+
+
+== Performance Test Validations Best Practices
+To apply performance testing consider the following best practices.
+
+=== Use a Controlled Environment
+Use a self-controlled environment to obtain repeatable, reliable and consistent data.
+To control and stabilize the environment:
+
+* Use an infrastructure made by a triplet:
+** Use a dedicated host for the Mule application to have isolated information preventing other running processes from interfering with the results in case of on-premises tests.
+** Use a separate and dedicated host for running load client tools that can drive the load without becoming the bottleneck.
+** Use a separate and dedicated host to run backend services like database, Tomcat, JMS, ActiveMQ, and so on. Use the same host the application uses in the production environment.
+** For on-premises tests, have the load client, runtime server and backend running in the same vpc to decrease network latency.
+** For Cloudhub tests, have the load client and backend server running on another dedicated machine but in the same region as the CloudHub worker to decrease network latency.
+* Use a wired stable network.
+* Don't use your laptop because it may outperform the same application deployed to a customer’s environment, particularly if it is virtualized.
+
+.Performance test infrastructure (Separate host avoid interference)
+image::mruntime-tuning-test-infrastructure.png[Performance test infrastructure]
+
+=== Choose the Right Tools
+
```

Comment:
The topic set introduction uses "testing tools" and this title refers to "the right tools," but the following sentence refers to "components" instead. I suggest we standardize on "tools," or make the distinction clearer, if there is one.